src/System.Collections.Specialized/src/System/Collections/Specialized/ListDictionary.cs

```diff
@@ -224,7 +224,7 @@ public bool Contains(object key)
         public void CopyTo(Array array, int index)
         {
             if (array == null)
-                throw new ArgumentNullException("array");
+                throw new ArgumentNullException("array", SR.ArgumentNull_Array);
```

Comment:
Does `ArgumentNull_Array` add enough value to justify keeping it and using it throughout? I wonder if this resource string should be removed altogether from this assembly. One less string resource to have to embed and localize.

The default message for `new ArgumentNullException("array")` is:

```
Value cannot be null.
Parameter name: array
```

Versus `new ArgumentNullException("array", SR.ArgumentNull_Array)`:

```
Array cannot be null.
Parameter name: array
```

The latter is more precise, but the former seems "good enough" IMHO.